### ISSUE TITLE
Select autocomplete suggestion using keyboard not working

### ISSUE THREAD

**Issue by dboris** [olahol/react-tagsinput#100]
Using the autocomplete example on `example/index.html`, if you type the first few letters of a state and then use the down arrow key to choose the state and press Enter to select it, the actual tag created is not the suggestion, but the letters typed so far. Selecting the state suggestion with the mouse works as expected.

**Comment by mrbongiolo** [comment]
I had the same problem, my current workaround is something like this:

```javascript
let handleOnChange = (event, {newValue, method}) => {
        if(method === 'enter') {
          event.preventDefault()
        } else {
          props.onChange(event)
        }
      }

return (
        <Autosuggest
          inputProps={{...other, onChange: handleOnChange}}
          otherFields...
        />
      )
```

Then I had to change `react-tagsinput` itself and add this lil bit on top of the `handleKeyDown` function:
```javascript
if (e.defaultPrevented)
          return
```

**Comment by olahol** [comment]
I've added @mrbongiolo fix to version `3.13.2`. I've also fixed the broken auto suggest example in `example/index.js`. Thanks guys.
